src/optimize/base_optimizer.js

```diff
@@ -101,40 +135,24 @@ class BaseOptimizer {
 
       module: {
         loaders: [
-          {
-            test: /\.less$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!less${mapQPre}dumpLineNumbers=comments`
-            )
-          },
-          {
-            test: /\.scss$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!sass${mapQPre}`
-            )
-          },
-          { test: /\.css$/, loader: ExtractTextPlugin.extract('style', `css${mapQ}`) },
+          { test: /\.less$/, loader: makeStyleLoader('less') },
+          { test: /\.scss$/, loader: makeStyleLoader('sass') },
+          { test: /\.css$/, loader: makeStyleLoader() },
           { test: /\.jade$/, loader: 'jade' },
           { test: /\.json$/, loader: 'json' },
           { test: /\.(html|tmpl)$/, loader: 'raw' },
-          { test: /\.png$/, loader: 'url?limit=10000&name=[path][name].[ext]' },
-          { test: /\.(woff|woff2|ttf|eot|svg|ico)(\?|$)/, loader: 'file?name=[path][name].[ext]' },
-          { test: /[\/\\]src[\/\\](core_plugins|ui)[\/\\].+\.js$/, loader: `rjs-repack${mapQ}` },
+          { test: /\.png$/, loader: 'url' },
```

Comment:
Is that the issue you intended to link to? I am not seeing the correlation.

The limit should bundle assets under the specified limit, as opposed to loading them as external assets. In master, I am seeing all assets being external - as expected when the query strings are being ignored. With this PR I am seeing smaller assets being included in the bundles. Where are we defining the limit now? By default it should have no limit and all assets should be external.